rocky/tools/view_helpers.py

```diff
@@ -179,3 +179,13 @@ def schedule_task(request: HttpRequest, organization_code: str, task: QueuePrior
                 "It may take some time, a refresh of the page may be needed to show the results."
             ),
         )
+
+
+def is_future_date(request: HttpRequest, observed_at: datetime) -> bool:
+    is_future_date = observed_at > convert_date_to_datetime(datetime.now(timezone.utc))
+    if is_future_date:
+        messages.error(
+            request,
+            _("You can't generate a report for an OOI on a date in the future."),
+        )
+    return is_future_date
```

Comment:
I suggest leaving the error out of this. This makes the function more generic and not strictly related to OOIs

Reply:
The idea here was to not have different messages across different views. I will make the message itself more generic.

Reply:
I think it is confusing when a function called `is_future_date` has a side effect of setting an error. In my eyes functions started with `is_` imply that they check something and return the result of that check and not cause side effects.

If the goal is to have the same error message we can assign the error string to a variable and use that in multiple places.